src/components/TableHeadCell.js

```diff
@@ -110,7 +112,11 @@ class TableHeadCell extends React.Component {
       <TableCell className={cellClass} scope={'col'} sortDirection={ariaSortDirection}>
         {options.sort && sort ? (
           <Tooltip
-            title={options.textLabels.body.toolTip}
+            title={
+              options.textLabels.body.columnHeaderTooltip
+                ? options.textLabels.body.columnHeaderTooltip(column)
+                : options.textLabels.body.toolTip
```

Comment:
I think `toolTip` is intended to be the default fallback option, right? If so, I think we'll need to add it to the default `textLabels.js` file or this errors out when `columnHeaderTooltip` isn't provided.

I think we'll also need to add documentation around proving `columnHeaderTooltip` in the localization section. Unless I'm misunderstanding, in which case, let me know.

Reply:
No, you're right. Forgot to clean up the PR :) my bad.

Reply:
No worries! Default is still missing from `textLabels.js`.

Reply:
Looks like this is working now, I may have been missing something locally last time I found this problem.

Reply:
Good to go then? Or do I still need to add something to textLabels.js?

Reply:
Nope, we're good to go.